src/digitalIdentity-frontend/src/app/shared/pop-up/edit-window-pop-up/edit-window-pop-up.component.html

```diff
@@ -14,72 +14,82 @@
-            <mat-datepicker
-              #picker
-              startView="year"
-              [startAt]="startDate"
-            ></mat-datepicker>
-          </mat-form-field>
+            <!-- Date block for date of birth -->
+            <mat-form-field *ngSwitchCase="'birthday'">
+              <input
+                matInput
+                [matDatepicker]="picker"
+                placeholder="Date of birth"
+                formControlName="{{ pi.key }}"
+                [min]="minDate"
+                [max]="maxDate"
+              />
+              <!-- formControlName defines the form Control entry where the value is saved -->
+              <mat-hint>DD/MM/YYYY</mat-hint>
+              <mat-datepicker-toggle
+                matSuffix
+                [for]="picker"
+              ></mat-datepicker-toggle>
+              <mat-datepicker
+                #picker
+                startView="year"
+                [startAt]="startDate"
+              ></mat-datepicker>
+            </mat-form-field>
 
-          <div *ngSwitchCase="'hr_employee'">
-            <mat-checkbox formControlName="{{ pi.key }}"
-              >HR Employee</mat-checkbox
-            >
-          </div>
+            <div *ngSwitchCase="'hr_employee'">
+              <mat-checkbox formControlName="{{ pi.key }}"
+                >HR Employee</mat-checkbox
+              >
+            </div>
 
-          <!-- default block -->
-          <mat-form-field *ngSwitchDefault>
-            <mat-label>{{ pi.label }}</mat-label>
-            <input matInput type="text" formControlName="{{ pi.key }}" />
-          </mat-form-field>
+            <!-- default block -->
+            <mat-form-field *ngSwitchDefault>
+              <mat-label>{{ pi.label }}</mat-label>
+              <input matInput type="text" formControlName="{{ pi.key }}" />
+            </mat-form-field>
+          </div>
         </div>
-      </div>
 
-      <button
-        mat-raised-button
-        [disabled]="!formGroup.valid"
-        color="primary"
-        (click)="editButtonEvent()"
-      >
-        Save
-      </button>
-      <button mat-raised-button color="primary" (click)="cancelButtonEvent()">
-        Cancel
-      </button>
-      <div *ngIf="!formFilled">Not all required form fields are filled!</div>
+        <button
+          mat-raised-button
+          [disabled]="!formGroup.valid"
+          color="primary"
+          (click)="editButtonEvent()"
+        >
+          Save
+        </button>
+        <button mat-raised-button color="primary" (click)="cancelButtonEvent()">
+          Cancel
+        </button>
+      </div>
+      <div *ngIf="!isIdValid()">
+        Data is not in the local database!
+        <button mat-raised-button color="primary" (click)="cancelButtonEvent()">
+          Cancel
+        </button>
+      </div>
+      <div *ngIf="isIdValid() && formFilled">
+        Not all required form fields are filled!
+      </div>
```

Comment:
I would whish such big changes in a basic component (not related to the proof templates in first instance) in a new PR in the future :)